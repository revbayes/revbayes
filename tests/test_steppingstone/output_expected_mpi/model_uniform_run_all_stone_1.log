Iteration	Posterior	Likelihood	Prior	alpha	bl[1]	bl[2]	bl[3]	bl[4]	bl[5]	bl[6]	bl[7]	bl[8]	bl[9]	bl[10]	bl[11]	bl[12]	bl[13]	bl[14]	bl[15]	bl[16]	bl[17]	bl[18]	bl[19]	bl[20]	bl[21]	bl[22]	bl[23]	bl[24]	bl[25]	bl[26]	bl[27]	bl[28]	bl[29]	bl[30]	bl[31]	bl[32]	bl[33]	bl[34]	bl[35]	bl[36]	bl[37]	bl[38]	bl[39]	bl[40]	bl[41]	bl[42]	bl[43]	er[1]	er[2]	er[3]	er[4]	er[5]	er[6]	gamma_rates[1]	gamma_rates[2]	gamma_rates[3]	gamma_rates[4]	pi[1]	pi[2]	pi[3]	pi[4]	pinvar	TL
0	-21299.58	-21266.95	-32.63586	0.7773872	0.16974	0.168662	0.210999	0.133256	0.168619	0.313303	0.444811	0.144602	0.223911	0.143098	0.259409	0.247554	0.173401	0.228823	0.176167	0.171195	0.179464	0.204297	0.2317	0.29045	0.356301	0.249263	0.598978	0.130801	0.0596818	0.00350073	0.199522	0.0878372	0.0678397	0.280063	0.311554	0.0843041	0.12464	0.0015487	0.0943049	0.0518897	0.0713799	0.0937782	0.024454	0.0249884	0.216217	0.205443	0.274405	0.028427	0.302733	0.0512674	0.0236699	0.558884	0.0350193	0.0907387	0.397868	0.950487	2.56091	0.36548	0.332428	0.0825843	0.219508	0.361375	7.896156
10	-21305.82	-21280.68	-25.14114	0.8422498	0.138072	0.156852	0.178031	0.103368	0.146279	0.239379	0.297048	0.125608	0.266767	0.145648	0.326862	0.210385	0.175072	0.243204	0.208453	0.134544	0.0958583	0.221015	0.286065	0.31696	0.427995	0.180697	0.686726	0.0724366	0.0419055	0.00202657	0.117408	0.0394518	0.0495578	0.252764	0.198613	0.0695436	0.0471969	0.0043555	0.0546304	0.0777044	0.0285255	0.0673677	0.026044	0.0266694	0.216771	0.202371	0.253456	0.0332712	0.293644	0.0587565	0.0210215	0.542105	0.0512023	0.104512	0.423604	0.968008	2.50388	0.355896	0.331849	0.0922128	0.220042	0.3758073	7.159688
