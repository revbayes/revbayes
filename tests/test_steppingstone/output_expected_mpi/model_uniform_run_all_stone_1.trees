Iteration	Posterior	Likelihood	Prior	psi
0	-21299.58	-21266.95	-32.63586	(Galeopterus_variegatus[&index=23]:0.598978,((((Lepilemur_hubbardorum[&index=11]:0.259409,(Microcebus_murinus[&index=14]:0.228823,Cheirogaleus_major[&index=4]:0.133256)[&index=35]:0.094305)[&index=39]:0.024454,((Varecia_variegata_variegata[&index=22]:0.249263,Lemur_catta[&index=10]:0.143098)[&index=38]:0.093778,Propithecus_coquereli[&index=19]:0.231700)[&index=40]:0.024988)[&index=27]:0.199522,((Loris_tardigradus[&index=12]:0.247554,Nycticebus_coucang[&index=15]:0.176167)[&index=37]:0.071380,(Perodicticus_potto[&index=18]:0.204297,(Otolemur_crassicaudatus[&index=16]:0.171195,Galago_senegalensis[&index=8]:0.144602)[&index=24]:0.130801)[&index=29]:0.067840)[&index=41]:0.216217)[&index=28]:0.087837,(Tarsius_syrichta[&index=21]:0.356301,(((Hylobates_lar[&index=9]:0.223911,Pan_paniscus[&index=17]:0.179464)[&index=32]:0.084304,(Colobus_guereza[&index=6]:0.313303,(Chlorocebus_aethiops[&index=5]:0.168619,Macaca_mulatta[&index=13]:0.173401)[&index=25]:0.059682)[&index=42]:0.205443)[&index=30]:0.280063,((Aotus_trivirgatus[&index=1]:0.169740,(Saimiri_sciureus[&index=20]:0.290450,Cebus_albifrons[&index=3]:0.210999)[&index=26]:0.003501)[&index=33]:0.124640,Callicebus_donacophilus[&index=2]:0.168662)[&index=43]:0.274405)[&index=31]:0.311554)[&index=36]:0.051890)[&index=34]:0.001549,Daubentonia_madagascariensis[&index=7]:0.444811)[&index=44];
10	-21305.82	-21280.68	-25.14114	(Galeopterus_variegatus[&index=23]:0.686726,Daubentonia_madagascariensis[&index=7]:0.297048,((((Perodicticus_potto[&index=18]:0.221015,(Otolemur_crassicaudatus[&index=16]:0.134544,Galago_senegalensis[&index=8]:0.125608)[&index=24]:0.072437)[&index=29]:0.049558,(Nycticebus_coucang[&index=15]:0.208453,Loris_tardigradus[&index=12]:0.210385)[&index=37]:0.028526)[&index=41]:0.216771,(((Cheirogaleus_major[&index=4]:0.103368,Microcebus_murinus[&index=14]:0.243204)[&index=35]:0.054630,Lepilemur_hubbardorum[&index=11]:0.326862)[&index=39]:0.026044,(Propithecus_coquereli[&index=19]:0.286065,(Varecia_variegata_variegata[&index=22]:0.180697,Lemur_catta[&index=10]:0.145648)[&index=38]:0.067368)[&index=40]:0.026669)[&index=27]:0.117408)[&index=28]:0.039452,(Tarsius_syrichta[&index=21]:0.427995,((Callicebus_donacophilus[&index=2]:0.156852,(Saimiri_sciureus[&index=20]:0.316960,(Cebus_albifrons[&index=3]:0.178031,Aotus_trivirgatus[&index=1]:0.138072)[&index=26]:0.002027)[&index=33]:0.047197)[&index=43]:0.253456,(((Chlorocebus_aethiops[&index=5]:0.146279,Macaca_mulatta[&index=13]:0.175072)[&index=25]:0.041905,Colobus_guereza[&index=6]:0.239379)[&index=42]:0.202371,(Pan_paniscus[&index=17]:0.095858,Hylobates_lar[&index=9]:0.266767)[&index=32]:0.069544)[&index=30]:0.252764)[&index=31]:0.198613)[&index=36]:0.077704)[&index=34]:0.004356)[&index=44];
