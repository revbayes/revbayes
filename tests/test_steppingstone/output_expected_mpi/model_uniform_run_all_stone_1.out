state	power	likelihood
4	0.59049	-21291.8
5	0.59049	-21269.2
6	0.59049	-21271
7	0.59049	-21272.4
8	0.59049	-21275.2
9	0.59049	-21278.7
10	0.59049	-21280.7
